hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkAdminHandlerTestBase.java

```diff
@@ -364,29 +367,29 @@ protected void givenValidAccount(
 
     protected void givenFungibleTokenRelation() {
         fungibleTokenRelation = TokenRelation.newBuilder()
-                .tokenNumber(tokenId.tokenNum())
-                .accountNumber(accountNum)
+                .tokenId(fungibleTokenId)
+                .accountId(accountId)
                 .balance(1000L)
                 .frozen(false)
                 .kycGranted(false)
                 .deleted(false)
                 .automaticAssociation(true)
-                .nextToken(0L)
-                .previousToken(3L)
+                .nextToken(asToken(0L))
```

Comment:
If asToken took shard and realm, this could be `asToken(0L, accountId.shard(), accountId.realm())`  Not much different, and not locked into 0 for both shard and realm.